jobs/github_tag_teams.groovy

```diff
@@ -1,29 +0,0 @@
-import util.Plumber
-
-def p = new Plumber(name: 'release/codekit/github-tag-teams', dsl: this)
```

Comment:
Similarly, this is used in the nightly/weekly/official releases for auxiliary repos.  Don't remove it.

Reply:
Actually, it looks like this job is not used, as the release jobs invoke the same code directly, not via the job.  Removal is OK.